### ISSUE TITLE
batch의 의미와 용도에 대해 알고싶습니다. (4장 MNIST)

### ISSUE THREAD

**Issue by Leeyoohyuk** [KHU-Dcom-Study/2018-1-DeepLearning-Study#32]
batch_size = 100
total_batch = int(mnist.train.num_examples / batch_size)

**Comment by Leeyoohyuk** [comment]
batch size는 stochastic Gradient Descent에서 데이터를 읽어 올 때
한 번에 읽어올 데이터의 갯수를 설정해주는 것입니다.
결과적으로 모든 데이터를 얻어오는 것은 같지만
이 사이즈에 따라 단위 계산 시간이 달라지며 총 계산시간에서도 차이가 날 수 있습니다.
